README.md

```diff
@@ -0,0 +1,52 @@
+## How to build
+1. Fork and clone the repositry
+2. Open the project folder and open a terminal within the project's directory.
+3. Use the command `premake5 vs2022`
+   - You can change the files generated by using a different [action](https://premake.github.io/docs/using-premake) than `vs2022`.   
+5. Open the new `Build` folder and double click the `AssetConverter.sln` file.
+6. Set the build configuration to `Release`.
+7. Use `ctrl`+`shift`+`b` to build the solution or right click the solution and select `Build Solution`
+8. The solution builds to `../Build/Bin/AssetConverter/[BuildConfig]`
+
+## How to use
+1. Download the `community-listfile.csv` from [here](https://github.com/wowdev/wow-listfile) and rename it to `listfile`.
+2. Go to your `../World of Warcraft/_classic"` folder and copy the `listfile.csv` and `AssetConverter.exe` to this folder.
+3. Run the `AssetConverter.exe`. If this is the first time running it, close the terminal.
+4. A new file named `AssetConverterConfig` has been created. Open it.
+    - If this is not your first time running the Asset Converter and you don't want to extract certain files, set `"Enabled:"` to `false` to the relavant value(s).
+    - If this is your first time, make sure your file looks like this:
+  ```json
+  {
+    "General": {
+        "Version": "0.4",
+        "ThreadCount": 19,
```

Comment:
ThreadCount here should be 0 by default, it will force multi threading. Unfortunately right now with how the config is auto generated on first launch, we set the ThreadCount to Cores - 1 rather than 0 which effectively does the same

Reply:
Do you want it -1 or 0?

Reply:
-1 is fine as well, might be better to reduce confusion

Reply:
Fixed